core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/CoarseMesosSchedulerBackend.scala

```diff
@@ -225,9 +225,22 @@ private[spark] class CoarseMesosSchedulerBackend(
             .addResources(createResource("cpus", cpusToUse))
             .addResources(createResource("mem",
               MemoryUtils.calculateTotalMemory(sc)))
-            .build()
+
+          sc.conf.getOption("spark.executor.docker.image").map { image:String =>
+            val container = task.getContainerBuilder()
+            val volumes = sc.conf
+              .getOption("spark.executor.docker.volumes")
+              .map(MesosSchedulerBackendUtil.parseVolumesSpec _)
+            val portmaps = sc.conf
+              .getOption("spark.executor.docker.portmaps")
```

Comment:
it seems that these are currently only intended to be used in mesos mode. Do you think it makes more sense to put them under the `spark.mesos.executor.docker.*` namespace?